src/main/webapp/components/workspace/workspace.js

```diff
@@ -138,121 +144,123 @@ export default () => {
 
   const { title } = attributes
   return (
-    <div
-      style={{
-        display: 'flex',
-        height: 'calc(100vh - 64px)',
-        overflow: 'hidden',
-      }}
-    >
+    <WorkspaceContext.Provider value={attributes}>
```

Comment:
The only thing that changed below this line is the closing tag of WorkspaceContext.Provider